app/controllers/recipes_controller.rb

```diff
@@ -7,10 +7,12 @@ def index
   end
 
   # GET /recipes/1 or /recipes/1.json
-  def show; end
+  def show
+    @recipefoods = RecipeFood.where(recipe_id: params[:id])
```

Comment:
When I clicked in the name of the ingredient in a recipe ( goes to recipe_food SHOW view ) I got a n1+ problem alert:

![image](https://user-images.githubusercontent.com/99205303/219174715-40c290f9-d069-4cea-8356-981110e01511.png)

Also check recipe foods :
- new view
- index view 
- edit view

Reply:
Thanks @tiagomarin for reviewing working on my issues and the fantastic suggestions. I forgot checking `recipe_foods` component. I will work on it now.

Reply:
Hi @tiagomarin . I have made changes based on the suggestions you gave me. Kindly review my code again please.
Thanks